city_scrapers/spiders/il_corrections.py

```diff
@@ -0,0 +1,157 @@
+import re
+from collections import defaultdict
+from datetime import datetime
+from io import BytesIO, StringIO
+
+import scrapy
+from city_scrapers_core.constants import ADVISORY_COMMITTEE
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from pdfminer.high_level import extract_text_to_fp
+from pdfminer.layout import LAParams
+
+
+class IlCorrectionsSpider(CityScrapersSpider):
+    name = "il_corrections"
+    agency = "Illinois Department of Corrections Advisory Board"
+    timezone = "America/Chicago"
+    start_urls = [
+        "https://www2.illinois.gov/idoc/aboutus/advisoryboard/Pages/default.aspx"
+    ]
+
+    def __init__(self):
+        self.links = {}
```

Comment:
Small thing but this doesn't need to be set on initialization since it'll be overwritten on the call to `parse` anyway

Reply:
Oh interesting, I thought it had to be instantiated at the outset. thanks!